docs/howtos/building-a-rust-component.md

```diff
@@ -150,12 +148,12 @@ data itself - often in a `meta` table.
 All logic for knowing which records need to be sync must be part of the
 application logic, and will often be implemented using `triggers`. It's quite
 common for components to use a "change counter" strategy, which can be
-summaried as:
+summarized as:
 
 * Every table which defines the "top level" items being synced will have a
   column called something like 'sync_change_counter' - the app will probably
   track this counter manually instead of using a trigger, because sync itself
-  will need different behaviour when it updates the records.
+  will need different behavior when it updates the records.
```

Comment:
hah :) "behaviour" is English and Australian spelling (but I agree we should use US spelling everywhere)

Reply:
I blame this completely on the extension I have installed on vscode that checks for typos 😇